### DISCUSSION TITLE
Feature Request: Use 'page back and forward' buttons on mouse to switch between workspaces.

### DISCUSSION THREAD

**Discussion by lhforsythe** [zen-browser/desktop#2481] [Ideas]
Hi there.

One feature, albeit small, I miss from Arc Browser is that of the ability to switch between workspaces via the back and forward page buttons on my mouse (the two buttons on the side that move you back and forward on a website). In Arc, I am able to hover over the tab bar and, rather than controlling the website, pressing these buttons would move me back and fourth through my workspaces.

Also, although a bit off topic, is there any possibility that a feature similar to Arc's Boosts would be implemented in Zen browser? I would love this considering that the Windows version of Arc has yet to implement this.

Thanks!

**Comment by bnwa** [comment]
+1'ing

**Comment by 12th-devs** [comment]
You can by clicking on the vertical tabs section and then pressing the buttons

Edit: fix typo

**Reply by bnwa** [reply]
> You can by clicking on the vertical tabs section and then pressing the buttons
> 
> Edit: fix typo

For posterity in case anyone else has my setup, if you've been using the SaneSideButtons app to get consistent side button navigation behavior on macOS without having to use the malware Logitech app, Zen Browser and SaneSideButtons don't play nice together and you'll need to enable the 'Ignore Zen Browser' option from the menu bar menu.

**Reply by 12th-devs** [reply]
Oh, I see
